### PR TITLE
Disable animation

### PR DESCRIPTION
**Pull request by litso**
Allow disabling animations on additional router interactions

### CODE REVIEW

File: Sources/TiltUp/Architecture/Router.swift [hunk 1]
```diff
@@ -47,7 +47,7 @@ public extension Router {
     }
 
     // MARK: Replacing root
-    func replaceRoot(with viewController: UIViewController, popHandler: (() -> Void)? = nil) {
+    func replaceRoot(with viewController: UIViewController, animated: Bool = true, popHandler: (() -> Void)? = nil) {
```
Comment (MichaelM1177) [inline Sources/TiltUp/Architecture/Router.swift:50]: looks like you aren't using this in the function, 

and nit, maybe also add for `removeAll` while adding for the others

**Review by MichaelM1177 — APPROVED** [review]

File: Sources/TiltUp/Architecture/Router.swift (hunk 1 above)
Comment (litso) [reply Sources/TiltUp/Architecture/Router.swift:50]: thanks for the catch. added it to the function and added animation to removeAll as well

Comment (MichaelM1177) [reply Sources/TiltUp/Architecture/Router.swift:50]: 👌, realizing that in `removeAll` it was `animated: false` before, so maybe that should still be the default there (or there is no reason for it to ever be true?), similar question with `navigationController.popToRootViewController(animated: false)` in this one, just confirming it should always be false?

Comment (litso) [reply Sources/TiltUp/Architecture/Router.swift:50]: Changes have been addressed.

**Pull request merged**